code/game/objects/items/holy_weapons.dm

```diff
@@ -390,8 +392,29 @@
 	lefthand_file = 'icons/mob/inhands/weapons/polearms_lefthand.dmi'
 	righthand_file = 'icons/mob/inhands/weapons/polearms_righthand.dmi'
 	name = "unholy pitchfork"
-	w_class = WEIGHT_CLASS_NORMAL
+	w_class = WEIGHT_CLASS_BULKY
 	desc = "Holding this makes you look absolutely devilish."
 	attack_verb = list("poked", "impaled", "pierced", "jabbed")
+	slot_flags = SLOT_BACK
```

Comment:
You add a new slot to carry the thing, but you don't add a sprite for doing so. Is there one already?

Reply:
There’s no sprite for another back slot on the holy weapons list unless I’m just dumb and blind, and while I don’t agree with just piling up undone work on someone (myself), this solves the issue of having nowhere to carry it yet. There’s a precedent for items that are invisible in slots, particularly the belt for the children of claymores. I’ll make this a priority if possible?